docker-compose.yml

```diff
@@ -293,29 +319,36 @@ services:
   redis:
     image: redis:latest
     ports:
-      - "6379:6379"
+      - '6379:6379'
   digilocker-certificate-api:
     image: ghcr.io/sunbird-rc/sunbird-rc-digilocker-certificate-api
     volumes:
       - ./services/digilocker-certificate-api/config/:/go/config/
     ports:
-      - "8087:8087"
+      - '8087:8087'
     environment:
       KEYCLOAK_CLIENT_ID: admin-api
-      KEYCLOAK_CLIENT_SECRET: ${KEYCLOAK_SECRET}
+      KEYCLOAK_CLIENT_SECRET: 61fdeaf0-9654-4c12-b7ff-f7116c9e4fca
```

Comment:
why did you put secret here? plz revert

Reply:
Reverted! This change was made while testing the CLI.